src/Agent.Worker/TaskManager.cs

```diff
@@ -290,9 +291,7 @@ public sealed class ExecutionData
         [JsonIgnore]
         public List<HandlerData> All => _all;
 
-#if !OS_WINDOWS || X86
-        [JsonIgnore]
-#endif
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
FYI, This may work as you want, but you can also create a method called "ShouldSerializeAzurePowerShell" that returns a bool wether to serialize or not. I think that would be easier than these changes. 
More info: https://www.newtonsoft.com/json/help/html/ConditionalProperties.htm

Reply:
Oh, that's neat - thanks! (What I really want is to understand whether/why this matters at all - I'd prefer to just serialize it the same on every platform. But I'm unclear what would break.)